addons/sourcemod/scripting/superzombiefortress.sp

```diff
@@ -3762,6 +3777,18 @@ void UpdateZombieDamageScale()
 		// If there atleast 1 CP, set progress by amount of CP capped
 		if (iMaxCP > 0)
 			flProgress = float(iCurrentCP) / float(iMaxCP);
+			
+		//If the map is too big for the amount of CPs, progress incerases with time
+		if(g_fTimeProgress > flProgress)
+		{
+			//Failsafe : Cannot exceed current CP (and a half)
+			float flProgressMax = (float(iCurrentCP)+1.0) / float(iMaxCP);
```

Comment:
If all CP captured but round did not end (szf_manncoscience), max would be above 1.0 meaning things can break if flProgress is above 1.0. Bit rare to see, but can happen.